app/server/appsmith-plugins/restApiPlugin/src/main/resources/form.json

```
@@ -14,6 +14,47 @@
           "label": "Headers",
           "configProperty": "datasourceConfiguration.headers",
           "controlType": "KEY_VAL_INPUT"
+        },
+        {
+          "label": "Send Authentication Information Key (Do not edit)",
+          "configProperty": "datasourceConfiguration.properties[0].key",
+          "controlType": "INPUT_TEXT",
+          "hidden": true,
+          "initialValue": "isSendSessionEnabled"
+        },
+        {
+          "label": "Send Authentication Information",
```

Here as well. I suggest calling it a signature instead of authentication information
```suggestion
          "label": "Send Appsmith signature header (X-APPSMITH-SIGNATURE)",
```